### DISCUSSION TITLE
Client Application as MVC

### DISCUSSION THREAD

**Discussion by utsav-hub** [iayti/CleanArchitecture#33] [Q&A] [answered]
Can we create a MVC client application which will consume these APIs and if Yes then where should we add this project within this project structure or create a whole new project?

If I create a whole new project then can I be able to access application layer?

Please suggest I want to create a client app but not with angular or blazor need simple mvc project and I am confused where to add it?

**Comment by iayti** [comment, marked as answer]
Hi,

Client.WorkerService is just for example of the auto generated nswag api implementations. You can delete it and create new MVC client. Change the "namespace" and "output" value in the CleanArchitecture.Api project nswag.json.

![Api_Client](https://user-images.githubusercontent.com/30297580/118934195-c1835a80-b952-11eb-9958-e88c0abf4471.jpg)

Have a nice coding 😊

**Reply by utsav-hub** [reply]
Thank you for your valuable inputs.
